websites/P/ParamountPlus/presence.ts

```diff
@@ -0,0 +1,219 @@
+        PresenceData.smallImageKey = "smithsonian";
+      }
+
+    } else if (path.includes("/live")) {
+
+      vidMdTl = document.getElementsByClassName("video__player-area")[0].querySelector("h1");
+      const liveTitle = document.getElementsByClassName("video__player-area")[0].querySelector("span.subTitle").textContent;
+
+      PresenceData.smallImageKey = "live";
+      PresenceData.smallImageText = (await strings).live;
+      PresenceData.startTimestamp = elapsed;
+
+      PresenceData.details = "Watching WCBS Live";
+      PresenceData.state = liveTitle;
+
+      if (path.includes("/cbsn/")) {
+
+        PresenceData.details = "Watching CBSN News";
+        PresenceData.state = vidMdTl.textContent;
+
+      } else if (path.includes("/sports/")) {
+
+        PresenceData.details = "Watching CBS Sports";
+        PresenceData.state = vidMdTl.textContent;
+
+      } else if (path.includes("/etl/")) {
+
+        PresenceData.details = "Watching ET Live";
+        PresenceData.state = vidMdTl.textContent;
+
+      }
+
+    } else if (vidArea && path.includes("/video") || path.includes("/movies")) {
+      video = document.querySelector("video");
+      const jsonData = JSON.parse(document.querySelector('[type="application/ld+json"]').innerHTML);
+      if (vidArea) {
+        if (path.includes("/movies")) {
+
+          movTitle = jsonData.name;
+
+        } else if (path.includes("/video")) {
+
+          title = jsonData.partOfSeries.name;
+
+          seasonEpi = "S" + jsonData.partOfSeason.seasonNumber + ":" +
+            "E" + jsonData.episodeNumber + " " +
+            jsonData.name;
+        }
+
+        const content = seasonEpi,
+          timestamps = presence.getTimestamps(
+            Math.floor(video.currentTime),
+            Math.floor(video.duration)
+          ),
+          live = timestamps[1] === Infinity;
+        
+        if (movTitle) {
+          PresenceData.state = movTitle;
+          PresenceData.details = "Watching Movie:";
+        }
+
+        if (title) {
+          PresenceData.details = title;
+        }
+
+        if (content) {
+          PresenceData.state = content;
+        }
+
+        if (path.includes("/news/")) {
+          PresenceData.details = "Watching News Content";
+          PresenceData.state = jsonData.partOfSeries.name + ": " + jsonData.name;
+        }
+
+        PresenceData.smallImageKey = live ? "live" : video.paused ? "pause" : "play";
+        PresenceData.smallImageText = live ?
+          (await strings).live :
+          video.paused ?
+          (await strings).pause :
+          (await strings).play;
+        PresenceData.startTimestamp = live ? elapsed : timestamps[0];
+        PresenceData.endTimestamp = live ? undefined : timestamps[1];
+
+        if (video.paused) {
+          PresenceData.startTimestamp = undefined;
+          PresenceData.endTimestamp = undefined;
+        }
+      }
+    }
+
+    presence.setActivity(PresenceData, video ? !video.paused : true);
```

Comment:
``setActivity(PresenceData)`` you don't use slideshow so you don't need to pass second argument.